processes/ingest/chicago_isac.py

```diff
@@ -72,18 +79,22 @@ def storePDFManifest(self, record):
                     'application/webpub+json',
                     flags
                 ])
+
                 record.has_part.insert(0, linkString)
-                #The second element of the has_part array will always be an array of pdfURLS from the json file
-                #Those elements of the array are popped out and appended back into the has_part array as strings instead of an array
-                for i in range(1, len(record.has_part)):
-                    if len(record.has_part[i]) > 1:
-                        urlArray = record.has_part[i]
-                        record.has_part.pop(i)
-                        for elem in urlArray:
-                            record.has_part.append(elem)
-                    else:
-                        record.has_part[i] = ''.join(record.has_part[i])
+                self.hasPartArrayElemToStringElem(record)
+
                 break
+    
+    @staticmethod
```

Comment:
Hmm does this have to be a static method?

Reply:
Not necessarily but the object itself isn't changing so there isn't a reason for it to be an object class method.